Sources/Stats/py_stats.cc

```diff
@@ -46,12 +48,39 @@ void AddStatsModule(py::module m) {
         return s + ">";
       });
 
+  auto as_dict = [](const Stats& self) {
+    py::dict d;
+    d["Mean"] = self.mean;
+    d["Sigma"] = self.error_of_mean;
+    d["Variance"] = self.variance;
+    d["R"] = self.R;
+    d["TauCorr"] = self.correlation;
+    return d;
+  };
+
   py::class_<Stats>(subm, "Stats")
       .def_readonly("mean", &Stats::mean)
       .def_readonly("error_of_mean", &Stats::error_of_mean)
       .def_readonly("variance", &Stats::variance)
       .def_readonly("tau_corr", &Stats::correlation)
-      .def_readonly("R", &Stats::R);
+      .def_readonly("R", &Stats::R)
+      .def("__repr__",
+           [](const Stats& self) {
+             std::ostringstream stream;
+             const double imag = self.mean.imag();
+             // clang-format off
+             stream << std::setprecision(4)
```

Comment:
Can we set the number of significant digits according to the error in self.error_of_mean? (the error itself only needs 2 significant digits at most, I would say)

Reply:
Alright, let's try and do it properly then.